test_data/rules/experiments/constructor_tearoffs/rules/always_specify_types.dart

```diff
@@ -10,8 +10,11 @@ import 'package:meta/meta.dart';
 
 /// Constructor tear-offs
 void constructorTearOffs() {
-  List.filled; // LINT
   List<List>.filled; // LINT
+  // todo(pq): should be OK.
+  // List<E> Function<E>(int, E) filledList = List.filled; // OK - generic function
```

Comment:
I wonder, yet again, whether it would be worthwhile to have a way to mark a linter test as failing so that we didn't need to remember to occasionally check to see whether this is passing yet. Maybe something like `OK-FAILING` and `LINT-FAILING`.

Reply:
I was thinking similarly.  @asashour filed #2815 and even has a PR!  I'll take a closer look later today.